internal/connect.go

```diff
@@ -33,16 +33,22 @@ func CallClusterOperation(config *hazelcast.Config, operation string, state *str
 	var err error
 	obj := NewRESTCall(config, operation, state)
 	params := obj.params
-	url := obj.url
+	urlStr := obj.url
```

Comment:
Let's try to use short variable names if possible

Reply:
it conflicts with url package, instead I thought changing the variable name would be better

Reply:
OK, that makes sense.